### PR TITLE
#408 Fixed NPE in ConstantUsageCheck

### PR DESCRIPTION
**Pull request by longtimeago**
#408 Fixed NPE in ConstantUsageCheck

### CODE REVIEW

**Comment by davvd** [comment]
@longtimeago Thanks, let me find someone who can review this pull request

**Comment by davvd** [comment]
@carlosmiranda review this one, please

File: qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/ConstantUsageCheck/Invalid.java [hunk 1]
```diff
@@ -1,9 +1,17 @@
+import org.apache.commons.lang.CharEncoding;
 /**
  * This is not a real Java class. It won't be compiled ever. It is used
  * only as a text resource in integration.ChecksIT.
  */
 public final class Invalid {
     private static final Invalid INSTANCE = new Invalid();
+    private static final int[] OTHER_INSTANCE =
```
Comment (carlosmiranda) [inline qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/ConstantUsageCheck/Invalid.java]: @longtimeago Let's define the array in a single line here

File: qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/ConstantUsageCheck/Valid.java [hunk 2]
```diff
@@ -6,6 +5,13 @@
  */
 public final class Valid {
     private static final Invalid INSTANCE = new Invalid();
+    private static final int[] OTHER_INSTANCE =
+        {
```
Comment (carlosmiranda) [inline qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/ConstantUsageCheck/Valid.java]: @longtimeago Same here, let's define the array in one line.

File: qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/ConstantUsageCheck/Invalid.java [hunk 3]
```diff
@@ -1,9 +1,17 @@
+import org.apache.commons.lang.CharEncoding;
```
Comment (carlosmiranda) [inline qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/ConstantUsageCheck/Invalid.java]: @longtimeago Is this import necessary?

Comment (longtimeago) [inline qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/ConstantUsageCheck/Invalid.java]: @carlosmiranda yep, left from experiments ... removing

File: qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/ConstantUsageCheck/Invalid.java (hunk 1 above)
Comment (longtimeago) [inline qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/ConstantUsageCheck/Invalid.java]: @carlosmiranda you are right. Works as expected even in one line. Fixed

**Comment by longtimeago** [comment]
@carlosmiranda fixed, please, take a look ones again

**Comment by carlosmiranda** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by davvd** [comment]
@carlosmiranda **16 mins** added to your account (payment number `60337470`), many thanks for your contribution! 23 hours and 45 mins spent here.; you're getting extra minutes here (c=1); +16 added to your rating, at the moment it is: [+3354](http://www.netbout.com/b/35023?open=rating)

**Comment by davvd** [comment]
@rultor please deploy

**Pull request merged**